app/src/controllers/attachments/sagas.js

```diff
@@ -0,0 +1,81 @@
+import { takeLatest, call, put, all } from 'redux-saga/effects';
+import { URLS } from 'common/urls';
+import { fetch } from 'common/utils';
+
+import { showModalAction } from '../../controllers/modal';
+import {
+  GET_ATTACHMENT_HAR,
+  ATTACHMENT_HAR_FETCH_ERROR,
+  GET_ATTACHMENT_BINARY,
+  ATTACHMENT_BINARY_FETCH_ERROR,
+  GET_ATTACHMENT_IMAGE,
+  ATTACHMENT_IMAGE_FETCH_ERROR,
+} from './constants';
+
+function fetchImageData({ projectId, binaryId }) {
+  return new Promise(() => ({
+    image: URLS.getFileById(projectId, binaryId),
+  }));
+}
+
+function fetchRealData({ projectId, binaryId }) {
```

Comment:
Coul you please clarify what does it mean 'real data' in this context? Why not `fetchData`, `fetchAttachment` or similar?

Reply:
I'll rename it to `fetchData` as suggested. It was meant to distinguis actual calls which turn to a real service eventaually (not like the image type since it's returning a promise to comply with the API).